refinery/ui/source/styles/file-browser.less

```diff
@@ -173,7 +173,7 @@
 // Tool Launch Table
 #tool-launch-panel {
   height: 84vh;
-  padding: 0 0 0 .5em;
+  padding: 0 .5em 0 .5em;
```

Comment:
Why did this need to be adjusted?

Reply:
With:
![screen shot 2017-10-24 at 5 02 04 pm](https://user-images.githubusercontent.com/5629547/31967860-3136e182-b8dd-11e7-847b-deb5acac078e.png)

Without:
![screen shot 2017-10-24 at 5 02 25 pm](https://user-images.githubusercontent.com/5629547/31967880-3a8eb0de-b8dd-11e7-8ed0-0a9621aea3c6.png)